cmake/config/options.cmake

```diff
@@ -10,37 +10,45 @@
 option( EVE_USE_PCH             "Build unit tests with precompield headers"     ON  )
 option( EVE_BUILD_TEST          "Build unit tests for eve"                      ON  )
 option( EVE_BUILD_RANDOM        "Build random precision tests for eve"          OFF )
+option( EVE_BUILD_INTEGRATION   "Build CMake integration tests for eve"         OFF )
 option( EVE_BUILD_BENCHMARKS    "Build benchmarks for eve"                      OFF )
 option( EVE_BUILD_DOCUMENTATION "Compile Doxygen documentation"                 ON  )
 
 if( EVE_USE_PCH )
-  set(pch_status "ON")
+  set(pch_status "ON ")
```

Comment:
why the space ?

Reply:
The word *OFF* has three characters, so the additional space on "ON " aligns the leading parenthesis on the following console messages.

Reply:
wont it affect the way we test for such option or did I accindentally wrote the correct thing ?

Reply:
They didn't appear to be used anywhere outside of `options.cmake`.

Reply:
Yeah noticed right after.